src/wallet/rpc/spend.cpp

```diff
@@ -1295,7 +1295,7 @@ RPCHelpMan send()
                     ParseOutputs(outputs),
                     InterpretSubtractFeeFromOutputInstructions(options["subtract_fee_from_outputs"], outputs.getKeys())
             );
-            CMutableTransaction rawTx = ConstructTransaction(options["inputs"], request.params[0], options["locktime"], rbf);
+            CMutableTransaction rawTx = ConstructTransaction(options["inputs"], request.params[0], options["locktime"], rbf, 3);
```

Comment:
You are changing the default version without motivation here. Also, as mentioned in previous review comments, please try to avoid magic numbers - here and in 3 other places.